[Refactor] 템플릿 디테일 설정

Chominsung0823 opened the pull request:
## 🪺 개요 
템플릿의 UI 통일, 불필요한 아이콘 제거, 권한 설정

## 💻 작업 사항 
- 기능이 없는 아이콘들을 제거하고, UI적으로 개선하였습니다.
- 권한 설정을 구체화하였고, 403 페이지가 아닌 메시지 출력 후 접근을 막았습니다.

## 🌱 Issue Number
- #73 

## ✅ 셀프 체크리스트
- [ ] PR 제목을 형식에 맞게 작성했나요 ?
- [ ] 브랜치 전략에 맞는 브랜치에 PR을 올리고 있나요 ?
- [ ] 테스트는 잘 통과했나요 ?
- [ ] 빌드에 성공했나요 ?
- [ ] 본인을 Assign 해주세요.
- [ ] 리뷰할 팀원들을 Request 해주세요.
- [ ] 해당 PR에 맞는 label을 붙여주세요.

## 🙏 To Reviewers
> 코드 리뷰보다는 화면단에서 시험하는 게 나을 거 같아요 수정이 필요한 부분이 있으면 바로 말씀해주세요~

Chominsung0823 committed:
Merge branch 'feat/settlement-templates-#69' into refactor/ui-detail-#73

Chominsung0823 committed:
refactor: 불필요한 아이콘 제거 (#73)
- 불필요한 Actions 아이콘 제거
- 각 리스트의 테이블 UI 일관성 유지

Chominsung0823 committed:
refactor: 상담 부분 디테일 설정 (#73)
- 필터링 설정
- 사용하지 않는 Team Members 삭제
- 의미없는 아이콘 제거

Chominsung0823 committed:
refactor: 모든 리스트 ui 일치(#73)
- 모든 리스트에 필터링 설정, 페이지네이션 설정
- 각 리스트를 내림차순으로 설정
- 테이블의 행을 클릭했을 때, 상세보기 페이지로 이동

Chominsung0823 committed:
feat: 소장 법률 검토 AI 접목(#73)
- review-result.html을 통해 소장 검토 결과를 볼 수 있음.

Chominsung0823 committed:
feat: 상담 부분 UI 통일(#73)
- 상담 노트 작성 기능
- 상담 완료 시 상담 시작 버튼 삭제, 상담 상세보기 버튼 추가

Chominsung0823 committed:
feat: 정산 부분 권한 설정(#73)
- 리스트 조회: 회계팀, 사무관
- 정산서 작성: 사무관
- 정산 상태 변경: 회계팀

Chominsung0823 committed:
feat: 정산 부분 권한 설정(#73)
- 정산 리스트 조회: 회계, 사무관
- 정산 상태 변경: 회계팀
- 정산서 작성: 사무관

Chominsung0823 committed:
refactor: 헤더 및 사이드바 불필요한 아이콘 제거(#73)
- 헤더: messages, boom boom 아이콘 제거
- 헤더: 로그인한 사용자 드롭다운에 로그아웃만 남기고 모두 제거
- 사이드바: 톱니바퀴 아이콘 제거

Chominsung0823 committed:
feat: 권한이 없는 페이지 접속 시 메시지 출력(#73)
- 기존 403 페이지 대신 권한이 없다는 메시지 출력

yangjunmo131 reviewed (commented):
옆에 있었던 수희님 얘기도 들어봐야겠지만 8주차 회의에서 제가 기억하기로는 INSPINIA 로그를 지우라고 하셨던 것 같아요. 그리고 프로필 사진도 카톡 등에 있는 기본 프로필로 바꿀수있으면 바꾸는게 나을 것 같습니다. 그리고 소장에서 ai 법률검토를 누르면 아래 오류가 뜨는데 한번 확인해주시면 감사하겠습니다! 시간없으시면 제가 내일 저녁 이후에 확인해볼게요 수고하셨습니다!{"isSuccess":false,"code":"PETITION_NOT_FOUND","message":"소장이 존재하지 않습니다.","results":{"validationErrors":null}}

Chominsung0823 committed:
refactor: 소장이 존재할 때만 법률검토 버튼 활성화(#73)
- 소장 작성 및 저장 후 AI 법률 검토 버튼 활성화
- AI 검토 시 로딩화면 표시

Chominsung0823 committed:
refactor: 변호사 배정 시 ID 입력에서 드롭다운 선택으로 변경(#73)

Chominsung0823 committed:
refactor: 소장이 존재할 때만 법률검토 버튼 활성화(#73)
- 소장 작성 및 저장 후 AI 법률 검토 버튼 활성화
- AI 검토 시 로딩화면 표시
- 기본 프로필 변경
- INSPINIA -> 해일로 변경

Chominsung0823 commented:
> 옆에 있었던 수희님 얘기도 들어봐야겠지만 8주차 회의에서 제가 기억하기로는 INSPINIA 로그를 지우라고 하셨던 것 같아요. 그리고 프로필 사진도 카톡 등에 있는 기본 프로필로 바꿀수있으면 바꾸는게 나을 것 같습니다. 그리고 소장에서 ai 법률검토를 누르면 아래 오류가 뜨는데 한번 확인해주시면 감사하겠습니다! 시간없으시면 제가 내일 저녁 이후에 확인해볼게요 수고하셨습니다!{"isSuccess":false,"code":"PETITION_NOT_FOUND","message":"소장이 존재하지 않습니다.","results":{"validationErrors":null}}

소장이 없어서 해당 로그가 뜨는 거 같아요. 그래서 소장이 없을 때 법률검토 버튼을 누를 시 소장 작성 유도 메시지를 넣어서 해결했습니다. INSPINIA -> 해일로 변경했습니다. 
그리고 변호사 배정 시 ID 입력하는 것을 드롭다운 형태로 바꾸어 변호사를 선택할 수 있도록 하였습니다.

Chominsung0823 committed:
Merge branch 'develop' into refactor/ui-detail-#73

yangjunmo131 commented:
> > 옆에 있었던 수희님 얘기도 들어봐야겠지만 8주차 회의에서 제가 기억하기로는 INSPINIA 로그를 지우라고 하셨던 것 같아요. 그리고 프로필 사진도 카톡 등에 있는 기본 프로필로 바꿀수있으면 바꾸는게 나을 것 같습니다. 그리고 소장에서 ai 법률검토를 누르면 아래 오류가 뜨는데 한번 확인해주시면 감사하겠습니다! 시간없으시면 제가 내일 저녁 이후에 확인해볼게요 수고하셨습니다!{"isSuccess":false,"code":"PETITION_NOT_FOUND","message":"소장이 존재하지 않습니다.","results":{"validationErrors":null}}
> 
> 소장이 없어서 해당 로그가 뜨는 거 같아요. 그래서 소장이 없을 때 법률검토 버튼을 누를 시 소장 작성 유도 메시지를 넣어서 해결했습니다. INSPINIA -> 해일로 변경했습니다. 그리고 변호사 배정 시 ID 입력하는 것을 드롭다운 형태로 바꾸어 변호사를 선택할 수 있도록 하였습니다.

제가 저장을 안누르고 바로 검토하려해서 그런거였네요. 작성유도메시지와 프로필 바뀐것도 확인했습니다! 로그인할 때는 INSPINIA 로고가 안사라진것같아서 시간날 때 확인한번 부탁드립니다! 추가로 사이드바를 닫으면 로고가 남아있는데 이것도 없애야겠습니다. 이건 큰 건 아닌데 해일말고 Hae:il로 바꾸는것도 괜찮아보입니다!
<img width="500" height="300" alt="image" src="https://github.com/user-attachments/assets/3fda43b8-83c6-4fea-8c34-2af871a2c142" />

<img width="400" height="250" alt="image" src="https://github.com/user-attachments/assets/f3838ead-30cd-40b0-8630-a0ed5afc9be1" />

erika0915 reviewed (approved):
로고 바꿔달라고 하셔서,, 그 부분만 적용해주시면 될 것 같아요 ~ 고생하셨습니당 !!

yangjunmo131 committed:
fix: 사건 완료처리 버튼 누락 (#73)

yangjunmo131 committed:
fix: 로고 교체 (#73)
